docs/design/overview.rst

```diff
@@ -168,7 +168,7 @@ Solitude
 Stores a limited amount of payment information and interacts with the payment
 provider.
 
-* *Documentation*: `solitude docs <https://solitude.readthedocs.org>`_
+* *Documentation*: `solitude docs <https://solitude.readthedocs.org/en/latest/>`_
```

Comment:
it does follow redirects so it showed a warning about this before I changed it. I hummed and hawed but I think it's probably best to use the target URL so that link checking is clean. However, it makes alternate locale requests slower since they get a redirect so that's kinda lame. That's more of RTD's problem.